cmd/skuba/cluster/upgrade.go

```diff
@@ -47,7 +47,7 @@ func newUpgradePlanCmd() *cobra.Command {
 		Use:   "plan",
 		Short: "Plan cluster upgrade",
 		Run: func(cmd *cobra.Command, args []string) {
-			clientSet, err := kubernetes.GetAdminClientSet()
+			clientSet, _, err := kubernetes.GetAdminClientSet()
```

Comment:
I suggest to add a new method `GetAdminClientSetWithConfig` or something similar, so we don't have to change all these function calls.

Reply:
In fact, is the rest config used anywhere in this PR?